certification/internal/shell/has_minimal_vulns_unshare.go

```diff
@@ -0,0 +1,75 @@
+package shell
+
+import (
+	"fmt"
+	"regexp"
+	"strings"
+
+	"github.com/redhat-openshift-ecosystem/openshift-preflight/certification"
+	"github.com/redhat-openshift-ecosystem/openshift-preflight/certification/errors"
+	log "github.com/sirupsen/logrus"
+)
+
+// HasMinimalVulnerabilitiesCheck evaluates the image to confirm that only vulnerabilities
+// below the Critical or Important severities, using oscap-podman and a corresponding
+// OVAL definition.
+type HasMinimalVulnerabilitiesUnshareCheck struct{}
+
+func (p *HasMinimalVulnerabilitiesUnshareCheck) Validate(image string) (bool, error) {
+	lines, err := p.getDataToValidate(image)
```

Comment:
Nit: I somewhat dislike `getDataToValidate` as a method name because it isn't descriptive. I don't have a suggestion at the moment, and I know we use it in multiple places, but If you can think of a way to change this so that we can somewhat establish a shared context with the reader/developer reviewing or attempting to follow this code, that'd be great.

No need to change others, we can change them as we come across them.

Reply:
I'll change this one to "scanImage", as that describes exactly what it is doing.